tests/integration/__init__.py

```diff
@@ -39,6 +41,7 @@
 from testtools.matchers import MatchesRegex
 
 from snapcraft.internal.os_release import OsRelease
+from snapcraft.internal import errors
```

Comment:
this is wrong, we should not import snapcraft into the integration tests.

Reply:
I fixed this in #2096 and re-based this one on top of it.